img/Apigouv_visual.svg

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+
+<svg
```

Comment:
“Destroys”? Weird. Thanks.

> 30% less only

Well, that's a pretty neat gain for #webperf, really  :wink: